make/test.mk

```diff
@@ -39,7 +75,7 @@ test/integration/versionchecker/testdata/test_manifests.tar: bin/scratch/oldcrds
 
 bin/scratch/oldcrds.tar: bin/scratch/git/upstream-tags.txt | bin/scratch/oldcrds
 	@# First, download the CRDs for all releases listed in upstream-tags.txt
-	<bin/scratch/git/upstream-tags.txt xargs -I% --max-procs=5 \
+	<bin/scratch/git/upstream-tags.txt xargs -I% -P5 \
```

Comment:
comment: I don't understand this change; it seems significantly less readable. Is `--max-procs` not available on some platforms?

Reply:
That's because `--max-procs` is specific to GNU and BSD xargs. It won't work on BusyBox xargs, which I used to test all of these makefiles 😅

Reply:
Thanks for the context :+1: